### PR TITLE
Add active boolean to sponsor_sellers

### PR DESCRIPTION
**Pull request by Jeremy-Xue**
Filter out inactive sponsorsellers (yifang)

### CODE REVIEW

**Review by justintmckibben — APPROVED** [review]

File: db/migrate/20200916000828_add_active_to_sponsor_seller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddActiveToSponsorSeller < ActiveRecord::Migration[6.0]
+  def change
+    add_column :sponsor_sellers, :active, :boolean
```
Comment (justintmckibben) [inline db/migrate/20200916000828_add_active_to_sponsor_seller.rb:3]: can you add default: true

Comment (justintmckibben) [reply db/migrate/20200916000828_add_active_to_sponsor_seller.rb:3]: and nullable: false

Comment (justintmckibben) [reply db/migrate/20200916000828_add_active_to_sponsor_seller.rb:3]: You can also do the migration here SponsorSellers.all.each { |s| s.update(active: true) }

Comment (Jeremy-Xue) [reply db/migrate/20200916000828_add_active_to_sponsor_seller.rb:3]: just realized this after making the pr, adding now!

Comment (Jeremy-Xue) [reply db/migrate/20200916000828_add_active_to_sponsor_seller.rb]: I think change_column_null :sponsor_sellers, :active, false, true also works, by setting nullable to false, and changing all null columns to true

**Pull request merged**